list.css

```diff
@@ -44,6 +44,10 @@ input { border-radius: 5px; }
     white-space: pre-wrap;
 }
 
+div.event_registration button:hover {
```

Comment:
The `:hover` state isn't necessary here, you can just to `div.event_registration button` -  https://developer.mozilla.org/en-US/docs/Web/CSS/cursor

Unless there's a reason for the `:hover`? I don't think it would make any difference.